packages/cli/src/lib/platform/authenticatedClient.spec.ts

```diff
@@ -121,4 +127,10 @@ describe('AuthenticatedClient', () => {
       })
     );
   });
+
+  it('#getUserInfo should return the user info', async () => {
+    const userInfo = await new AuthenticatedClient().getUserInfo();
```

Comment:
Related to my comment below, if `initialize` is needed on the platform client in order to correctly receive the user info, then this is what the UT should focus on.

Because, otherwise, this test is kind of useless, we mock the return and just check that it returns what we just mocked ;)